tests/std/tests/P0896R4_counted_iterator/test.cpp

```diff
@@ -302,6 +302,42 @@ struct instantiator {
     }
 };
 
+// Also test P2259R1 Repairing input range adaptors and counted_iterator
+struct simple_forward_iter {
+    using value_type        = double;
+    using difference_type   = long;
+    using iterator_category = input_iterator_tag;
+    using iterator_concept  = forward_iterator_tag;
+
+    value_type operator*() const;
+    simple_forward_iter& operator++();
+    simple_forward_iter operator++(int);
+
+    bool operator==(simple_forward_iter const&) const;

```

Comment:
Style nitpick: This file uses west `const` (like product code and most test code).